packages/sdk/src/testutils.spec.ts

```diff
@@ -3,53 +3,8 @@ import { Bech32, Encoding } from "@iov/encoding";
 
 import hackatom from "./testdata/contract.json";
 
-const { fromHex } = Encoding;
-
-export function leb128Encode(uint: number): Uint8Array {
-  if (uint < 0) throw new Error("Only non-negative values supported");
-  if (uint > 0x7fffffff) throw new Error("Only values in signed int32 range allowed");
-  const out = new Array<number>();
-  let value = uint;
-  do {
-    // eslint-disable-next-line no-bitwise
-    let byte = value & 0b01111111;
-    // eslint-disable-next-line no-bitwise
-    value >>= 7;
-
-    // more bytes to come: set high order bit of byte
-    // eslint-disable-next-line no-bitwise
-    if (value !== 0) byte ^= 0b10000000;
-
-    out.push(byte);
-  } while (value !== 0);
-  return new Uint8Array(out);
-}
-
-export function getRandomizedHackatom(): Uint8Array {
-  const data = Encoding.fromBase64(hackatom.data);
-
-  // The return value of the export function cosmwasm_api_0_6 is unused and
-  // can be randomized for testing.
-  //
-  // Find position of mutable bytes as follows:
-  // $ wasm-objdump -d contract.wasm | grep -F "cosmwasm_api_0_6" -A 1
-  // 0136d2 func[198] <cosmwasm_api_0_6>:
-  //  0136d3: 41 83 0c                   | i32.const 1539
-  //
-  // In the last line, the addresses [0136d3, 0136d3+1, 0136d3+2] hold a one byte instruction
-  // and a two byte value (leb128 encoded 1539). See also
-  // https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md#constants-described-here.
-
-  // Any unsigned integer from 128 to 16383 is encoded to two leb128 bytes
-  const min = 128;
-  const max = 16383;
-  const random = Math.floor(Math.random() * (max - min)) + min;
-  const bytes = leb128Encode(random);
-
-  data[0x0136d3 + 1] = bytes[0];
-  data[0x0136d3 + 2] = bytes[1];
-
-  return data;
+export function getHackatom(): Uint8Array {
```

Comment:
Much simpler.
Though I am still a bit sad to see that code go away - cool stuff but not really maintainable.
It will live forever in git though.

Reply:
Yeah I was feeling the same. Knowledge of those wasm internals will suvive and can easily be re-applied later on.